### PR TITLE
EPMRPP-42615: Exception handling

### PR DESCRIPTION
**Pull request by DiscoElevator**

### CODE REVIEW

File: app/src/components/containers/pageErrorBoundary/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2 2019 EPAM Systems
```
Comment (AmsterGet) [inline app/src/components/containers/pageErrorBoundary/index.js]: Should it be just `Copyright 2019 EPAM Systems`?

File: app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ import { activeProjectSelector } from 'controllers/user';
 import { showModalAction } from 'controllers/modal';
 import { SpinningPreloader } from 'components/preloaders/spinningPreloader';
 import { DASHBOARD_PAGE_EVENTS } from 'components/main/analytics/events';
-import { CHARTS, MULTI_LEVEL_WIDGETS_MAP, NoDataAvailable } from 'components/widgets';
+import { CHARTS, MULTI_LEVEL_WIDGETS_MAP, NoDataAvailable, ErrorMessage } from 'components/widgets';
```
Comment (AmsterGet) [inline app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx]: I think will be better to import it directly from `components/main/errorMessage`

File: app/src/components/widgets/index.js [hunk 3]
```diff
@@ -49,6 +49,8 @@ import {
   ComponentHealthCheck,
 } from './multiLevelWidgets';
 
+export { ErrorMessage } from '../main/errorMessage';
```
Comment (AmsterGet) [inline app/src/components/widgets/index.js]: And remove re-export here

File: app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx (hunk 2 above)
Comment (DiscoElevator) [reply app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx]: Oh, sure. Something weird has been left after moving components around

**Pull request merged**